.github/workflows/issues.yml

```diff
@@ -0,0 +1,32 @@
+name: Issues Bot
+
+on:
+  # Uncomment this PR trigger for testing.
+  # pull_request:
+  #   branches:
+  #   - master
+  schedule:
+  - cron: '5 16-23 * * 1,2,3,4,5'  # 5 past the hour, every hour from 16-24 UTC, on Monday - Friday
```

Comment:
Do we need it to run that often?  Why not just once per day?  I suppose it doesn't matter too much, if it's cheap to run.

Reply:
Yea, once a day probably makes more sense 👍